recipes/openfx/all/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+cmake_minimum_required(VERSION 3.1)
+project(openfx VERSION 1.4.0 LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+add_definitions(-D_HAS_AUTO_PTR_ETC)
+
+# Flags
+if(MSVC)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWINDOWS -DNOMINMAX")
```

Comment:
for the record, [add_definitions](https://cmake.org/cmake/help/latest/command/add_definitions.html) might be a better alternative over direct flags manipulations. no need to change it, just FIY.